reinforcement_learning/rl_cartpole_ray/rl_cartpole_ray.ipynb

```diff
@@ -0,0 +1,564 @@
+{
```

Comment:
<p>How long does the training takes? Usually we keep it 10~20 min</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>It took less than 3 minutes with my last few runs.</p>

Reply:
<p>Increased the training time finally by setting the <code>train_max_run</code>  param in RLEstimator to 500 and removing stopping critera from RLlib config.</p>